serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc

```diff
@@ -41,11 +77,8 @@ If Minikube does not work with the default driver, also known as `docker`, you c
 minikube start [...] --driver podman
 ----
 
-== Install the {product_name} Operator
```

Comment:
Since you mention in the note that the section to install minikube can be skipped, I think we should make sure the installation of the operator is in a separate section (otherwise people don't know how far they need to skip), so I suggest keeping this line, with === title.